multimedia/vapoursynth/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vapoursynth vapoursynth 51 R
```

Comment:
They released R52 after you submitted this PR, so you could update the port, but we can also accept it at version 51 if you want to handle the update later.

Reply:
Yep, updated to 52.